tests/ConstantTest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Lambdish\Phunctional\Tests;
+
+use PHPUnit_Framework_TestCase;
+use function Lambdish\Phunctional\constant;
+
+final class ConstantTest extends PHPUnit_Framework_TestCase
+{
+    /** @test */
+    public function it_should_return_a_value()
+    {
+        $value = 5;
+        $constantClosure  = constant($value);
```

Comment:
align

Reply:
May be it would be nice to add some kind of pre-commit hook and check coding style :thinking:

Reply:
In a future... 😆

Reply:
Is a future or a promise? :trollface: